README.md

```diff
@@ -36,96 +36,20 @@ _Tip: If you want to connect to the specific service outside of docker, then use
 
 ```mermaid
 erDiagram
-  Users ||--|{ Friends : requesterId
-  Users ||--|{ Friends : userId
-  Users ||--|{ Notifications : userId
-  Users }|--|{ Chats : id
-  Users }|--|{ Lessons : id
-
-  Users {
-    bigint Id PK
-    int country
-    int language
-    int timezone
-    nvarchar firstName
-    nvarchar lastName
-    int age
-    nvarchar email
-    nvarchar imagePath
-    int sex
-    int languageLevel
-    int status
-    boolean isSubscribed
-    boolean isBanned
+  User {
+      bigint Id
+      nvarchar Name
+      nvarchar Email
+      nvarchar PhoneCode
+      nvarchar PhoneNumber
+      nvarchar ImagePath
+      int Country
+      int Language
+      int TimeFormat
+      int DateFormat
+      boolean IsBanned
+      boolean IsDeleted
```

Comment:
Please, check what you commit) Sadly, this is for a lot of files, you make merging development incorrectly...